.github/workflows/upload_testflight.yml

```
@@ -39,22 +39,22 @@ jobs:
         uses: swift-actions/setup-swift@v2
         with:
             swift-version: "5.10"
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)